src/trustyai/visualizations/factory.py

```diff
@@ -0,0 +1,69 @@
+"""Visualizations.factory module"""
+# pylint: disable=import-error, wrong-import-order, protected-access, missing-final-newline
+from bokeh.io import show
+
+from trustyai.explainers import SHAPResults, LimeResults, pdp
+from trustyai.metrics.distance import LevenshteinResult
+from trustyai.visualizations.visualization_results import VisualizationResults
+from trustyai.visualizations.shap import SHAPViz
+from trustyai.visualizations.lime import LimeViz
+from trustyai.visualizations.pdp import PDPViz
+from trustyai.visualizations.distance import DistanceViz
+
+
+def get_viz(explanations) -> VisualizationResults:
+    """
+    Get visualization according to the explanation method
+    """
+    if isinstance(explanations, SHAPResults):
+        return SHAPViz()
+    if isinstance(explanations, LimeResults):
+        return LimeViz()
+    if isinstance(explanations, pdp.PDPResults):
+        return PDPViz()
+    if isinstance(explanations, LevenshteinResult):
+        return DistanceViz()
+    raise ValueError("Explanation method unknown")
+
+
+def plot(
+    explanations, output_name=None, render_bokeh=False, block=True, call_show=True
```

Comment:
Should we add type hints to these arguments (especially `explanations`)?

It would make easier for an user to know if they're passing an illegal object. An `Union` of the allowed types should be enough.